### ISSUE TITLE
Doesn't work.

### ISSUE THREAD

**Issue by eeshaa1** [rschouwenburg/trilium-theme-notion#1]
Not sure what else to say... followed the instructions and when I loaded the theme it just loaded Trilium's vanilla "Light" theme.

**Comment by rschouwenburg** [comment]
Were you able to select the theme in Options -> Appearance?

**Comment by eeshaa1** [comment]
yes, but it did not load as the theme in the screenshots, just trilium's light theme

**Comment by rschouwenburg** [comment]
Ok, if you got that far, the set up of the note is correct (type=CSS, atrribute #appTheme).

Which version of Trilium are you using? It works fine on 0.60.x but I have not tried 0.61 which is in beta.

**Comment by eeshaa1** [comment]
"App Version" says 0.60.4
